metrics/next-ios-render-test-runner-style.json

```diff
@@ -0,0 +1,11 @@
+{
+    "base_test_path": "integration",
+    "cache_path": "cache-style.db",
+    "expectation_paths": [
+        "expectations/platform-all"
+    ],
+    "ignore_paths": [
+        "ignores/platform-all.json",
+        "ignores/platform-macos.json"
+    ]
```

Comment:
Metrics would be added later?

Reply:
Yes, currently there are several failing test cases, the following steps would be making all tests passing(adding ios ignores, expectations), generating metrics.

Reply:
Now all tests are passed, platform dependent ignores and expectations are added. Metrics will be done in follow-up pr.